ibm/data_source_ibm_is_instance_network_interface.go

```diff
@@ -0,0 +1,448 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func dataSourceIBMIsInstanceNetworkInterface() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMIsInstanceNetworkInterfaceRead,
+
+		Schema: map[string]*schema.Schema{
+			"instance": &schema.Schema{
```

Comment:
I think across all VPC resource we used instance <to refer to the id> can you change to <instance_name> and <network_interface_name>